tests/rptest/clients/kafka_cli_tools.py

```diff
@@ -145,6 +161,46 @@ def produce(self,
         ]
         self._execute(cmd)
 
+    def list_acls(self):
+        args = ["--list"]
+        return self._run_strict("kafka-acls.sh", args)
+
+    def create_cluster_acls(self, username, op):
+        """
+        Add allow+describe+cluster ACL
+        """
+        args = ["--add"]
+        args += ["--allow-principal", f"User:{username}"]
+        args += ["--operation", op, "--cluster"]
+        return self._run_strict("kafka-acls.sh", args)
+
+    def _run_strict(self, script, args):
```

Comment:
should this be named `_run_with_acls()`  or I don't otherwise get the comment of looking at all users.

Reply:
it throws, the other doesn't

Reply:
ack